<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messenger</class>
 <widget class="QDialog" name="Messenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>912</width>
    <height>548</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>912</width>
    <height>548</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChatiX</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>510</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border: none;
    border-radius: 15px;
    }

QPushButton:pressed {
     background-color: rgb(180,180,180);
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/embleminput/send-message.png</normaloff>:/embleminput/send-message.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }


QListView::item:selected {
     border: none;
	background-color:rgb(152,180,180);
 }</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>671</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }

QListView::item:selected {
     border: none;
	background-color:rgb(152,180,180);
 }</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }</string>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Light</family>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Логин или уникальное имя группы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>621</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }</string>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }
    QPushButton:pressed {
    background-color:rgb(180,180,180);
    }</string>
   </property>
   <property name="text">
    <string>Добавить </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>91</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Light</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Список чатов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>3</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border: none;
    border-radius: 5px;
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }
    QPushButton:pressed {
    background-color:rgb(180,180,180);
    }</string>
   </property>
   <property name="text">
    <string>Создать группу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
